config/openshift/kustomization.yaml

```diff
@@ -6,7 +6,6 @@ namespace: openshift-operators
 # "wordpress" becomes "alices-wordpress".
 # Note that it should also match with the prefix (text before '-') of the namespace
 # field above.
-namePrefix: codeflare-operator-
```

Comment:
Could you move it to the webhook overlay.

Reply:
Deployment name and Webhook resources names show as expected. - `codeflare-operator-.....`